### ISSUE TITLE
It's called Terraform, not Regionform

### ISSUE THREAD

**Issue by spy-tech** [hashicorp/terraform#10832]
I feel the design decision to not support multiple regions with one invocation of Terraform is very undesirable. 

I have seen ticket https://github.com/hashicorp/terraform/issues/870 and the demand for this feature clearly exists. 

One way to resolve this is to attach to every resource a field specifying the region. Essentially the "region" is a bit of a global variable and in that sense "special", which we shouldn't have. I would be happy with such a solution.   

Another way to resolve this is to just rename Terraform to RegionForm ;) The name Terraform suggest it's a planetary level infrastructure management tool, which IMHO it cannot be called until this feature exists.

**Comment by stack72** [comment]
Hi @spy-tech 

Thanks so much for getting in touch - Terraform can be used to run against multiple regions. You would basically set your terraform configuration and accept a variable for region then you can loop over as many regions as you want

I understand the need for this feature (as per #870) but I am going to close this out as a duplicate. Please do add information to that issue so we can track it in a single area

Thanks

Paul

**Comment by spy-tech** [comment]
@stack72 Thanks for your reply.

The status of #870 is closed. The common interpretation of "closed" is that it is closed and that HashiCorp in this case will not care further for such tickets. As such, I would like you to reopen #870. That's the only reason I opened this ticket. Additionally, the other ticket is quite old, so it might have been that new support already arrived in the meanwhile.  Finally, looping over regions is not semantically the same as what I requested.
